packages/grafana-ui/src/components/PanelChrome/PanelContext.ts

```diff
@@ -9,18 +9,27 @@ export interface PanelContext {
   /** Dashboard panels sync */
   sync?: DashboardCursorSync;
 
+  /** The dashboard/panel timezone */
+  timeZone: TimeZone;
+
   /**
    * Called when a component wants to change the color for a series
    *
    * @alpha -- experimental
    */
   onSeriesColorChange?: (label: string, color: string) => void;
 
+  /**
+   * Called when a series should be toggled
+   *
+   * @alpha -- experimental
+   */
   onToggleSeriesVisibility?: (label: string, mode: SeriesVisibilityChangeMode) => void;
 }
 
 export const PanelContextRoot = React.createContext<PanelContext>({
   eventBus: new EventBusSrv(),
+  timeZone: DefaultTimeZone,
```

Comment:
There is a `getTimeZone()` function that you can use instead since `DefaultTimeZone` might not be the currently selected default for the user.